### PR TITLE
Display error when retrieving resource and refactors load and save actions.

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -145,40 +206,49 @@ export const setUnusedRDF = (state, action) => {
...
-  SHOW_GROUP_CHOOSER: showGroupChooser,
   CLOSE_GROUP_CHOOSER: closeGroupChooser,
-  SHOW_RESOURCE_TEMPLATE_CHOOSER: showResourceTemplateChooser,
   CLOSE_RESOURCE_TEMPLATE_CHOOSER: closeResourceTemplateChooser,
+  COPY_NEW_RESOURCE: copyResourceToEditor,
+  ITEMS_SELECTED: setItemsOrSelections,
   LANGUAGE_SELECTED: setMyItemsLang,
-  SHOW_RDF_PREVIEW: showRdfPreview,
+  LANGUAGES_RECEIVED: languagesReceived,
+  LOADING_LANGUAGES: loadingLanguages,
```
Comment (jcoyne) [inline src/reducers/index.js]: I wonder if we should change these to `RETRIEVE_LANGUAGES_STARTED/FINISHED` for rconsistency.

File: src/actions/index.js [hunk 2]
```diff
@@ -119,61 +111,62 @@ export const retrieveResourceTemplateStarted = resourceTemplateId => ({
...
+export const setResourceTemplate = resourceTemplate => ({
+  type: 'SET_RESOURCE_TEMPLATE',
+  payload: resourceTemplate,
 })
 
-export const toggleCollapse = reduxPath => ({
-  type: 'TOGGLE_COLLAPSE',
-  payload: { reduxPath },
+export const setResourceTemplateSummary = resourceTemplateSummary => ({
+  type: 'SET_RESOURCE_TEMPLATE_SUMMARY',
```
Comment (jcoyne) [inline src/actions/index.js]: Maybe `RESOURCE_TEMPLATE_SUMMARY_LOADED`? Not `SET_`

File: src/actions/index.js [hunk 3]
```diff
@@ -119,61 +111,62 @@ export const retrieveResourceTemplateStarted = resourceTemplateId => ({
...
+  payload: resourceTemplateSummary,
 })
 
-export const clearResourceTemplates = () => ({
-  type: 'CLEAR_RESOURCE_TEMPLATES',
+export const setSaveResourceError = (uri, reason) => ({
+  type: 'SET_SAVE_RESOURCE_ERROR',
```
Comment (jcoyne) [inline src/actions/index.js]: `SAVE_RESOURCE_ERROR` not `SET_`

**Comment by justinlittman** [comment]
Various renaming completed @jcoyne .

**Pull request merged**